### PR TITLE
Fix deflection base charges

### PR DESCRIPTION
**Pull request by StudentAlleg**
## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested my changes.

### CODE REVIEW

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/Deflection.java [hunk 1]
```diff
@@ -26,6 +27,7 @@
 
 @Singleton
 @BPvPListener
+@Slf4j
```
Comment (StudentAlleg) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/Deflection.java:30]:
```suggestion
```

File: champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/Deflection.java [hunk 2]
```diff
@@ -2,6 +2,7 @@
 
 import com.google.inject.Inject;
 import com.google.inject.Singleton;
+import lombok.extern.slf4j.Slf4j;
```
Comment (StudentAlleg) [inline champions/src/main/java/me/mykindos/betterpvp/champions/champions/skills/skills/knight/passives/Deflection.java:5]:
```suggestion
```

File: champions/src/main/resources/configs/skills/skills.yml [hunk 3]
```diff
@@ -285,10 +285,15 @@ skills:
       minBonusDamage: 0.5
     deflection:
       enabled: true
-      maxlevel: 3
+      maxlevel: 5
```
Comment (Mykindos) [inline champions/src/main/resources/configs/skills/skills.yml:288]: passives have a max rank of 3

**Review by Mykindos — CHANGES_REQUESTED** [review]

File: champions/src/main/resources/configs/skills/skills.yml (hunk 3 above)
Comment (StudentAlleg) [reply champions/src/main/resources/configs/skills/skills.yml]: oops, dont know why that was there

**Review by Mykindos — APPROVED** [review]

**Pull request merged**